### DISCUSSION TITLE
DevTools failed to load source map: Could not load content for http://localhost:10101/Applications/F1 MultiViewer.app/Contents/Resources/app/.webpack/renderer/main_window/preload.js.map

### DISCUSSION THREAD

**Discussion by weljajoh** [f1multiviewer/issue-tracker#10] [Q&A] [answered]
I'm not having much success. Signing in on the welcome screen briefly flashes the 'sign into F1TV account' button and then nothing. The developer view shows these errors:

```
DevTools failed to load source map: Could not load content for http://localhost:10101/Applications/F1 MultiViewer.app/Contents/Resources/app/.webpack/renderer/main_window/preload.js.map: HTTP error: status code 404,
net::ERR_HTTP_RESPONSE_CODE_FAILURE
TrackStatusLabel.tsx:70 Uncaught (in promise) Error: Error invoking remote method 'f1tv:auth:get-tokens': No handler registered for 'f1tv:auth:get-tokens'

```
Intel Macbook Air with macOS 11.6.6 
Same problem is seen on a 2020 M1 Mac Mini with macOS 12.4. If I try to access the failing URL in the browser I get a nicely formatted '404 | The requested path could not be found' response so there is something listening on port 10101.

Issue persists which local macOS firewall and pihole on the nework disabled.

**Comment by f1multiviewer** [comment]
> DevTools failed to load source map: Could not load content for http://localhost:10101/Applications/F1 MultiViewer.app/Contents/Resources/app/.webpack/renderer/main_window/preload.js.map: HTTP error: status code 404,
net::ERR_HTTP_RESPONSE_CODE_FAILURE

This can be safely ignored, source maps are just for mapping processed/minified code to the original source, but the line below indicates that the preload script wasn't loaded correctly. Honestly, not sure why. I noticed you're still on Big Sur, but since you reproduced it on Monteray too, I guess we can rule that out. It's weird though that since nobody else has this issue, that you can reproduce it on two separate machines 😕

**Reply by f1multiviewer** [reply]
I'll add some code to the next build that should make it easier to debug what's happening in the preload script!

**Comment by weljajoh** [comment, marked as answer]
This turns out to have been a local configuration issue. Although I was sure I disabled the pihole while having trouble, it wasn't. Looking at the pihole query log showed it blocking lookups of clients2.google.com. Whitelisting that resolved the issue.
